operations/http.go

```diff
@@ -259,7 +259,11 @@ func (ac *legacyClient) GetConfig(versionId string) (*model.Project, error) {
 		return nil, NewAPIError(resp)
 	}
 	ref := &model.Project{}
-	if err := util.ReadYAMLInto(resp.Body, ref); err != nil {
```

Comment:
does [this](https://github.com/evergreen-ci/evergreen/blob/943224c9b4d87ee64af38f8ae946004330ec9a16/operations/http.go#L246) need to be changed as well?

Reply:
yup, fixed